### PR TITLE
i think i did it right

### PR DESCRIPTION
**Pull request by wolfplay013**
d

### CODE REVIEW

File: commands/rtsitem.lua [hunk 1]
```diff
@@ -17,7 +17,7 @@ function command.run(message, mt)
 
   local uj2 = dpf.loadjson(uj2f, defaultjson)
   
-  local item = 'hauntedgrass'
+  local item = 'sparecryopod'
   local itemtype = 'cons'
```
Comment (HastagGuigui) [inline commands/rtsitem.lua:21]: [BREAKING ISSUE] Spare Cryopod is not a consumable, please change it to something that isn't `'cons'`

Comment (HastagGuigui) [reply commands/rtsitem.lua:21]: Side note: `itemtype` (what needs to be changed to fix the issue) only affects the actual item, as rtsitem gives both an item and its essence. The item is either an item or a consumable, while the essence is always a consumable, no matter what.

Comment (wolfplay013) [reply commands/rtsitem.lua:21]: fixing

**Comment by wolfplay013** [comment]
fixed it! please check if possible

**Comment by HastagGuigui** [comment]
yep! if anyone who can merge it sees this please merge #56 as well as they both rely on eachother pretty much (one's for the actual item's implementation, the other for the !rtsitem part)

**Pull request merged**